Patches/Vanilla Apparel Expanded - Accessories/Apparel_Utility.xml

```diff
@@ -112,6 +112,30 @@
 				</value>
 			</li>
 
+			<!-- == Ranged Shield Pack == -->
+
+			<Operation Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="VAEA_Apparel_RangedShieldBelt"]/statBases</xpath>
+				<value>
+					<Bulk>4</Bulk>
+					<WornBulk>2</WornBulk>
+				</value>
+			</Operation>
+
+			<Operation Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="VAEA_Apparel_RangedShieldBelt"]/statBases/EnergyShieldRechargeRate</xpath>
+				<value>
+					<EnergyShieldRechargeRate>0.195</EnergyShieldRechargeRate>
+				</value>
+			</Operation>
+
+			<Operation Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="VAEA_Apparel_RangedShieldBelt"]/statBases/EnergyShieldEnergyMax</xpath>
+				<value>
+					<EnergyShieldEnergyMax>1.35</EnergyShieldEnergyMax>
```

Comment:
What's the reason for this change, and what's the impact on balance in combat?

Reply:
Ranged shield belts were left unpatched and with the patched values of vanilla shield belts, the gap between them got much wider and ranged ones remained quite weak.